src/app/view/components/site-selector/dot-site-selector.component.spec.ts

```diff
@@ -38,39 +36,39 @@ describe('Site Selector Component', () => {
         ],
         providers: [
           {provide: SiteService, useValue: siteServiceMock},
-          IframeOverlayService
+          IframeOverlayService, PaginatorService
         ]
     });
 
     fixture = DOTTestBed.createComponent(SiteSelectorComponent);
     comp = fixture.componentInstance;
```

Comment:
These names are not descriptive, it should be componentInstance, debugElement and nativeElement